maps/southern_cross/loadout/loadout_head.dm

```diff
@@ -1,4 +1,9 @@
 /datum/gear/head/pilot
-	display_name = "helmet, pilot (Pilot)"
-	path = /obj/item/clothing/head/ompilot/alt //VOREStation Edit
+	display_name = "helmet, standard pilot (Pilot)"
+	path = /obj/item/clothing/head/pilot_vr //VOREStation Edit
+	allowed_roles = list("Pilot")
+
+/datum/gear/head/pilot
+	display_name = "helmet, colored pilot (Pilot)"
+	path = /obj/item/clothing/head/pilot_vr/alt //VOREStation Edit
```

Comment:
Copypaste error, also should probably be moved to a different file since they are entirely new items.